scripts/local/e2e_test.sh

```diff
@@ -33,9 +33,7 @@ ginkgo build ./tests/
 
 # Run the tests
 echo "Running e2e tests $RUN_E2E"
-RUN_E2E=true ./tests/tests.test \
-  --ginkgo.vv \
-  --ginkgo.label-filter=${GINKGO_LABEL_FILTER:-""}
+RUN_E2E=true REPO_ROOT=$TELEPORTER_PATH ginkgo -p tests/tests.test -vv
```

Comment:
Parallel runs require the ginkgo cli. This feature is not supported from the binary on its own.